miden-tx/src/tests/kernel_tests/test_tx.rs

```diff
@@ -579,3 +598,318 @@ fn test_build_recipient_hash() {
         "recipient hash not correct",
     );
 }
+
+#[test]
+fn test_load_foreign_account_basic() {
```

Comment:
This test is good - but I think we also need a tests which tests this functionality more comprehensively. Specifically, we make a foreign procedure call from within another account as a part of a complete transaction (we could skip authentication for this transaction). This may be quite a bit of work - so, let's create an issue for this.

Reply:
Created an issue: #918